# workaroung for musl getmntent: avoid using fusermount in test setup
--- a/tests/common.rb
+++ b/tests/common.rb
@@ -213,11 +213,5 @@ def nonroot_testenv(bindfs_args, options = {}, &block)
 end

 def umount_cmd
-    if !`which fusermount3`.strip.empty?
-        'fusermount3 -uz'
-    elsif !`which fusermount`.strip.empty?
-        'fusermount -uz'
-    else
-        'umount'
-    end
+    'umount'
 end

 def assert
